WalletWasabi/WabiSabi/Client/ArenaClient.cs

```diff
@@ -114,6 +114,31 @@ public async Task RegisterOutputAsync(Guid roundId, long value, Script scriptPub
 			WeightCredentialClient.HandleResponse(outputRegistrationResponse.WeightCredentials, realWeightCredentialResponseValidation);
 		}
 
+		public async Task ReissuanceAsync(Guid roundId, long value, IEnumerable<Credential> amountCredentialsToPresent)
+		{
+			Guard.InRange(nameof(amountCredentialsToPresent), amountCredentialsToPresent, 0, AmountCredentialClient.NumberOfCredentials);
```

Comment:
This is done by the WabiSabi client anyway.

Reply:
I added it because RegisterOutputAsync, ConfirmConnectionAsync also checking that.